cs3api4lab/auth/channel_connector.py

```diff
@@ -18,13 +18,19 @@ def __init__(self, **kwargs):
                 ca_cert = None
 
                 if config.client_cert is not None and len(config.client_cert) > 0:
-                    cert = open(config.client_cert, 'rb').read()
+                    client_cert_file = open(config.client_cert, 'rb')
```

Comment:
This is not safe. Opening a file might fail and it's never closed.

You should use the recommended way of opening files (surrounded with `with`, which automatically closes the file):
```python
with open(...) as f:
  x = f.read()
```